lib/check-tags.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+const log = require('./out')({});
+const _ = require('lodash');
+
+function checkTags(app, mod, name) {
+  // Returns true if the module should be skipped
+
+  if ((app.excludeTags && !mod.tags) || (!app.includeTags && !app.excludeTags)) {
+    return false; // No checks to run.
+  } else if (app.includeTags && !mod.tags) {
+    return true; // No tags for this module
+  }
+
+  if (typeof mod.tags === 'string') {
+    mod.tags = mod.tags.split(','); // If mod.tags is a single field
```

Comment:
I think `mod.tags = [mod.tags]` is enough here.
If it is intentional to accept multiple tags in a string, this should be documented and tested.